src/documents/IoCSS.md

```diff
@@ -47,74 +52,78 @@ fluid.uiOptions(".my-uio-container", {
+|`source`|(Mutually exclusive with `record`) An [IoC expression](IoCReferences.md) into the options structure of the source component, referencing what to copy to the target.|
+|&nbsp;&nbsp;`removeSource`|(Only possible if `source` is used) true/false: If true, the `source` options block is removed from its original site in the options structure when it is forwarded to the `target`.|
+|&nbsp;&nbsp;`exclusions`|(Only possible if `source` is used) A list of EL paths into the `source` material which should not be forwarded. Whether or not `removeSource` is used, these will be retained in their original position in the source component's options.|
+|`priority`|(Optional) a [Priority specification](Priorities.md) specifying how the priority of this distribution is to be resolved with respect to any other options distributions which are targetted at the same component in the tree as this one. Typically one will write `after:otherNamespace` to indicate that this distribution should be stronger than (should merge options on top of) another distribution whose `namespace` is `otherNamespace`, or `before:otherNamespace` to indicate that this distribution should be weaker than (have option merged on top of by) a distribution whose `namespace` is `otherNamespace`.|
+|`namespace`|(Optional, recommended) A `namespace` to identify this options distribution amongst others in an extended design. This can be used by the `priority` field of other distributions in order to defer to it or be deferred to by it. (The framework should also ensure to uniquify distributions with respect to namespace at the target - not currently implemented)|
+
+In the case that a hash of these records is provided, the keys of the structure will be interpreted as the `namespace` of the distribution.
 ## IoCSS Selectors ##
 
 Component matching rules:
 
 | Form | Description |
 |------|-------------|
-|*|matches any component - universal selector|
-|E|matches any component holding a context name of E - special support for the string "that" as with standard IoC context matching|
-|E#myid|matches any component with a context name of E with id equal to myid (of no use to developers since component ids cannot be predicted)|
+|`*`|matches any component - universal selector|
+|`/`|matches the **global root component** of the entire component tree - may only be used in the head position of an IoCSS selector|
+|`E` or `&E`|matches any component holding a context name of `E` - the `&` character may be omitted for the first context in a group. There is special support for the context `that` as with standard IoC context matching|
```

Comment:
It begins a selector segment which matches a context name - can you think of a way I could make this any more clear?